pynest/nest/tests/test_NodeCollection.py

```diff
@@ -518,7 +518,7 @@ def test_GetConnections(self):
         self.assertEqual(get_conn_some.get('source'), compare_source)
         self.assertEqual(get_conn_some.get('target'), compare_target)
 
-        compare_list = [3, 1, 0, 15, 6]
+        compare_list = [3, 1, 0, 18, 6]
```

Comment:
Here and in `test_synapsecollection.py` I had to change the synapse_id of the static connection from 15 to 18. Apparently, by adding a synapse model the IDs of the other synapses change which leads to failing tests.
Is there a way to automatically get the correct synapse id for the static connection instead of hard-coding it?

Reply:
@hakonsbm: could you comment on whether a hard-coded ID is OK here, or whether we need to dynamicaly get this from the kernel?

Reply:
@clinssen If all synapse IDs change when adding a synapse, we should get the synapse ID for reference values dynamically, so don't have to change the reference values every time a new synapse is added. The synapse ID of a given synapse model can be found in the synapsedict, so we could do something like this for example:
```python
expected_syn_model = 'static_synapse'
expected_syn_id = nest.ll_api.sli_func('synapsedict')[expected_syn_model]
compare_list = [3, 1, 0, expected_syn_id, 6]
```
And similarly in other places we use synapse IDs as reference values.

Reply:
I did what you suggested. It seems to work.